src/elements/content-sidebar/activity-feed/task-new/TaskStatus.js

```diff
@@ -19,26 +20,29 @@ type Props = {|
 |};
 
 const statusMessageKeyMap = {
-    [TASK_NEW_APPROVED]: messages.tasksFeedApprovedLabel,
-    [TASK_NEW_COMPLETED]: messages.tasksFeedCompletedLabel,
-    [TASK_NEW_REJECTED]: messages.tasksFeedRejectedLabel,
-    [TASK_NEW_NOT_STARTED]: messages.tasksFeedInProgressLabel,
-    [TASK_NEW_IN_PROGRESS]: messages.tasksFeedInProgressLabel,
+    [TASK_NEW_APPROVED]: messages.taskFeedApprovedUppercaseLabel,
+    [TASK_NEW_COMPLETED]: messages.taskFeedCompletedUppercaseLabel,
+    [TASK_NEW_REJECTED]: messages.taskFeedRejectedUppercaseLabel,
+    [TASK_NEW_NOT_STARTED]: messages.taskFeedInProgressUppercaseLabel,
+    [TASK_NEW_IN_PROGRESS]: messages.taskFeedInProgressUppercaseLabel,
+};
+
+const typeKeyMap = {
+    [TASK_NEW_APPROVED]: 'success',
+    [TASK_NEW_COMPLETED]: 'success',
+    [TASK_NEW_REJECTED]: 'error',
+    [TASK_NEW_NOT_STARTED]: 'default',
+    [TASK_NEW_IN_PROGRESS]: 'default',
 };
 
 const Status = React.memo<Props>(({ status }: Props) => (
-    <FormattedMessage
-        {...messages.tasksFeedStatusLabel}
-        values={{
-            taskStatus: (
-                <span className={`bcs-TaskStatus-message ${camelCase(status)}`}>
-                    <FormattedMessage {...statusMessageKeyMap[status]} />
-                </span>
-            ),
-        }}
-    >
-        {(...msg: Array<React.Node>): React.Node => <div className="bcs-TaskStatus">{msg}</div>}
-    </FormattedMessage>
+    <LabelPill.Pill type={typeKeyMap[status]}>
+        <LabelPill.Text>
+            <span className="bcs-TaskStatus">
```

Comment:
if the purpose of this class is to bold the text, we should remove this span and class style, and update the label pill to apply the text treatment, so there are fewer CSS overrides

Reply:
Good note.